src/property/types/CloudString.h

```diff
@@ -49,6 +49,12 @@ class CloudString : public Property {
     void clear() {
       _value = PropertyActions::CLEAR;
     }
+    const char * c_str() {
```

Comment:
Const-correctness might make sense.
```suggestion
    const char * c_str() const {
```

Reply:
this change was dropped